desktop/packaging/Linux/rs.ruffle.Ruffle.desktop

```diff
@@ -3,8 +3,8 @@ Type=Application
 Name=Ruffle
 GenericName=Flash Player
 Comment=Play Flash games & movies
-Icon=rs.ruffle.Ruffle
-Exec=ruffle %u
+Icon=icon
+Exec=ruffle_desktop
```

Comment:
> Changing these will also break Flatpak builds.

Yep, this will definitely be reverted when I rebase this PR. I was changing the values just to match the current ones: The executable cargo builds *is* named ruffle_desktop and the icons *are* named icon. Therefore I adapted the file to get working AppImages. But yeah, I'll change that when rebasing.

Reply:
The last two directories are actually removed in this PR.
But for all folders but `ISSUE_TEMPLATE`, there are no technical reasons to have their names upper case, it's just *convention*. The path of `Assets.xcassets` is just given to `actool`, it could have any other name. And the colorset folders again are just names listed in `Info.plist`. They could be lowercase.

The reason why those are uppercase is the same as why the platform names should be: convention. Going from the other side, the reason why most other folders have lower-case numbers is because those are module and crate names, which are lowercase by convention. They are not referencing any existing name that are capitalized in a specific way.

I don't see any reason to keep convention for all other folders, but break the convention in this specific case. It's most consistent and more readable to capitalize the OS name in their proper way and stick to the convention.

Reply:
We should refer to the product we ship as "ruffle" as opposed to "ruffle desktop", the binary and crate name internally is because this is something of a monorepo and we need the internal distinction. Users do not.

For directory names: I personally don't care enough to argue one way or another as long as we're functional and consistent. We refer to OS names as an identifier lowercase most places, it makes sense that directory names follow suite. The mac one is macOS with a capital right now because it's part of the app bundle, which is an Apple specification.

Reply:
Yes, I will change it back to `ruffle`, especially since that's relied upon by the (newer) Flathub integration. I'll revert this when I rebase it (I can do that in the next few days).